<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Пользователи</string>
        </attribute>
        <widget class="QTableView" name="tableViewUsers">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1091</width>
           <height>491</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Информация о пользователях</string>
        </attribute>
        <widget class="QTableView" name="tableView_Info">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>1091</width>
           <height>491</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Привелегии</string>
        </attribute>
        <widget class="QTableView" name="tableView_Priv">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1091</width>
           <height>491</height>
          </rect>
         </property>
        </widget>
        <zorder>tableView_Priv</zorder>
        <zorder>verticalLayoutWidget_3</zorder>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Цены</string>
        </attribute>
        <widget class="QTableView" name="tableView_Prices">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1091</width>
           <height>551</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ButtonAddUser">
         <property name="text">
          <string>Добавить пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonDelUser">
         <property name="text">
          <string>Удалить пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRegUser">
         <property name="text">
          <string>Редактировать пользователя</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ButtonChangePriv">
         <property name="text">
          <string>Изменить привелегии</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonCHangePrices">
         <property name="text">
          <string>Изменить цены</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonAddNews">
         <property name="text">
          <string>Добавить новость</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushButton_9</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
